src/utilities/embeddable-utils.test.ts

```diff
@@ -36,23 +36,26 @@ describe("Embeddable utility functions", () => {
 
     const exportableAnswer = getAnswerWithMetadata(interactiveState, embeddable) as IExportableOpenResponseAnswerMetadata;
 
-    expect(exportableAnswer).toBeDefined();
-
-    expect(exportableAnswer.remote_endpoint).toBe("");
-    expect(exportableAnswer.question_id).toBe("managed_interactive_123");
-    expect(exportableAnswer.question_type).toBe("open_response");
     expect(exportableAnswer.id).toBeDefined();          // random uuid
-    expect(exportableAnswer.type).toBe("open_response_answer");
-    expect(exportableAnswer.answer_text).toBe("test");
-    expect(exportableAnswer.answer).toBe("test");
-    expect(exportableAnswer.submitted).toBeNull();
-    const expectedReport = JSON.stringify({
-      mode: "report",
-      authoredState: `{"version":1,"questionType":"open_response","prompt":"<p>Write something:</p>"}`,
-      interactiveState: `{"answerType":"open_response_answer","answerText":"test"}`,
-      version: 1
+    // overwrite this id so we can do a full equals test below without a random value
+    exportableAnswer.id = "overwritten-answer-id";
```

Comment:
FYI, in CLUE I recently used an `omitDeep()` function to solve this problem:
```typescript
  expect(omitDeep(someObject, ["id"])).toEqual({ ... object without ids ... })
```
where `omitDeep()` looks like this
```typescript
// cf. https://github.com/lodash/lodash/issues/723#issuecomment-677383457
const omitDeep = (input: Record<string, unknown>, excludes: string[]): Record<string, unknown> => {
  return Object.entries(input).reduce((nextInput, [key, value]) => {
    const shouldExclude = excludes.includes(key);
    if (shouldExclude) return nextInput;

    if (Array.isArray(value)) {
      const arrValue = value;
      const nextValue = arrValue.map((arrItem) => {
        if (arrItem && typeof arrItem === "object") {
          return omitDeep(arrItem, excludes);
        }
        return arrItem;
      });
      nextInput[key] = nextValue;
      return nextInput;
    } else if (value && typeof value === "object") {
      nextInput[key] = omitDeep(value as Record<string, unknown>, excludes);
      return nextInput;
    }

    nextInput[key] = value;

    return nextInput;
  }, {} as Record<string, unknown>);
};

export default omitDeep;
```

I thought briefly about implementing it as a jest matcher so that one could use it like so:
```
  expect(someObject).toEqualOmitting(["id"], { ... obj without ids ...});
```
but decided it wasn't worth the extra effort at this point, but if it were being used across projects it might well be worth it.